### PR TITLE
Edit docs for style, naming, and broken links

### PR DESCRIPTION
**Pull request by theletterf**
This PR fixes several issues:

- Old naming / product identity
- Adds missing GDI spec files
- FIxes broken links
- Style improvements

### CODE REVIEW

File: website/docs/r/victor_ops_integration.html.markdown [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 sidebar_current: "docs-signalfx-resource-victor-ops-integration"
 description: |-
-  Allows Terraform to create and manage SignalFx VictorOps Integrations
+  Allows Terraform to create and manage Splunk On-Call Integrations
 ---
 
 # Resource: signalfx_victor_ops_integration
 
-SignalFx VictorOps integration.
+Splunk On-Call integrations.
```
Comment (benkeith-splunk) [inline website/docs/r/victor_ops_integration.html.markdown:11]: Is the name VictorOps completely retired now? It is kind of confusing how the resource is still called VictorOps but the name is "Splunk On-Call". I would maybe at least reference the old name in parenthesis for all of the devops people out there who don't keep up with the latest marketing updates.

Comment (theletterf) [reply website/docs/r/victor_ops_integration.html.markdown:11]: AFAIK, yes. Is that right, @mfoulds-splunk?

File: README.md [hunk 2]
```diff
@@ -1,76 +1,86 @@
...
+- Configured [GOPATH](http://golang.org/doc/code.html#GOPATH)
+- `$GOPATH/bin` added to your `$PATH`
 
-```sh
-$ make test
-```
+To compile the provider, run `make build`. This builds the provider and put its binary inside the `$GOPATH/bin` directory:
+
+   ```sh
+   $ make bin
```
Comment (bgola-signalfx) [inline README.md]: I know this is not your change but it should say `make build`

File: website/docs/index.html.markdown [hunk 3]
```diff
@@ -3,46 +3,42 @@ layout: "signalfx"
 page_title: "Provider: SignalFx"
 sidebar_current: "docs-signalfx-index"
 description: |-
-  The SignalFx provider is used to interact with the resources supported by SignalFx. The provider needs to be configured with the proper credentials before it can be used.
+  Use the Splunk Observability Cloud provider, formerly known as Splunk Observability Cloud provider, to interact with the resources supported by Splunk Observability Cloud. Configure the provider with the proper credentials before using it.
```
Comment (bgola-signalfx) [inline website/docs/index.html.markdown]: `...formerly known as SignalFx Terraform provider`?

File: website/docs/r/dashboard.html.markdown [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+A dashboard is a curated collection of specific charts and supports dimensional [filters](https://docs.splunk.com/observability/en/data-visualization/dashboards/dashboard-create-customize.html#customize-dashboard-filters), [dashboard variables](https://docs.splunk.com/observability/en/data-visualization/dashboards/dashboard-create-customize.html#customize-dashboard-variables) and [time range](https://docs.splunk.com/observability/en/data-visualization/use-time-range-selector.html) options. These options are applied to all charts in the dashboard, providing a consistent view of the data displayed in that dashboard. This also means that when you open a chart to drill down for more details, you are viewing the same data that is visible in the dashboard view.
 
-~> **NOTE** Since every dashboard is included in a [dashboard group](dashboard_group.html) (SignalFx collection of dashboards), you need to create that first and reference it as shown in the example.
+Since every dashboard is included in a [dashboard group](dashboard_group.html), which a collection of dashboards, you need to create that first and reference it as shown in the example.
```
Comment (bgola-signalfx) [inline website/docs/r/dashboard.html.markdown]: typo? `which **is** a`

File: website/docs/r/dashboard_group.html.markdown [hunk 5]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: "signalfx"
-page_title: "SignalFx: signalfx_dashboard_group"
+page_title: "Splunk Observability Cloud: signalfx_dashboard_group"
 sidebar_current: "docs-signalfx-resource-dashboard-group"
 description: |-
-  Allows Terraform to create and manage SignalFx Dashboard Groups
+  Allows Terraform to create and manage sashboard groups in Splunk Observability Cloud
```
Comment (bgola-signalfx) [inline website/docs/r/dashboard_group.html.markdown]: typo: dashboard

File: website/docs/r/detector.html.markdown [hunk 6]
```diff
@@ -119,34 +121,35 @@ Sends an email to every member of a team.
 notifications = ["TeamEmail,teamId"]
 ```
 
-### VictorOps
+### Splunk On-Call
```
Comment (bgola-signalfx) [inline website/docs/r/detector.html.markdown]:
```suggestion
### Splunk On-Call (formerly VictorOps)
```

File: README.md (hunk 2 above)
Comment (theletterf) [reply README.md]: Fixed!

File: website/docs/index.html.markdown (hunk 3 above)
Comment (theletterf) [reply website/docs/index.html.markdown]: +1

File: website/docs/r/dashboard.html.markdown (hunk 4 above)
Comment (theletterf) [reply website/docs/r/dashboard.html.markdown]: Good catch! Fixed

File: website/docs/r/dashboard_group.html.markdown (hunk 5 above)
Comment (theletterf) [reply website/docs/r/dashboard_group.html.markdown]: Fixed!

**Review by bgola-signalfx — APPROVED** [review]

File: website/docs/r/gcp_integration.html.markdown [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
+  Allows Terraform to create and manage GCP Integrations for Splunk Observability Cloud
 ---
 
 # Resource: signalfx_gcp_integration
 
-SignalFx GCP Integration
+Splunk Observability Cloud GCP Integration.
 
-~> **NOTE** When managing integrations, use a session token of an administrator to authenticate the SignalFx provider. See [Operations that require a session token for an administrator](https://dev.splunk.com/observability/docs/administration/authtokens#Operations-that-require-a-session-token-for-an-administrator). Otherwise you'll receive a 4xx error.
+~> **NOTE** When managing integrations, use a session token of an administrator to authenticate the Splunk Observability Cloud servability Cloud provider. See [Operations that require a session token for an administrator](https://dev.splunk.com/observability/docs/administration/authtokens#Operations-that-require-a-session-token-for-an-administrator). Otherwise you'll receive a 4xx error.
```
Comment (bgola-signalfx) [inline website/docs/r/gcp_integration.html.markdown]: `Cloud servability` - typo

File: website/docs/r/victor_ops_integration.html.markdown (hunk 1 above)
Comment (theletterf) [reply website/docs/r/victor_ops_integration.html.markdown:11]: Maureen confirmed it's OK if it's not something in the code.

**Comment by theletterf** [comment]
@bgola-signalfx Added changelog entry and fixed the typo!

File: website/docs/r/gcp_integration.html.markdown [hunk 8]
```diff
@@ -10,7 +10,7 @@ description: |-
 
 Splunk Observability Cloud GCP Integration.
 
-~> **NOTE** When managing integrations, use a session token of an administrator to authenticate the Splunk Observability Cloud servability Cloud provider. See [Operations that require a session token for an administrator](https://dev.splunk.com/observability/docs/administration/authtokens#Operations-that-require-a-session-token-for-an-administrator). Otherwise you'll receive a 4xx error.
+~> **NOTE** When managing integrations, use a session token of an administrator to authenticate the Splunk Observability Cloud Observability Cloud provider. See [Operations that require a session token for an administrator](https://dev.splunk.com/observability/docs/administration/authtokens#Operations-that-require-a-session-token-for-an-administrator). Otherwise you'll receive a 4xx error.
```
Comment (bgola-signalfx) [inline website/docs/r/gcp_integration.html.markdown]: duplicated name: `Observability Cloud Observability Cloud`

**Review by bgola-signalfx — APPROVED** [review]

**Pull request merged**